client/components/locations/dashboard/_index.html

```diff
@@ -21,7 +21,7 @@
     <div flex flex-gt-sm="{{fs.panel ? 100 : 100}}" layout="column">
       <div flex flex-gt-sm="{{fs.panel ? 100 : 100}}" layout="column" class="dashboard-charts">
         <div layout="" layout-wrap>
-          <div flex="100" flex-gt-md="{{fs.panel ? 100 : 33}}" iiing-hide="fs && fs.panel != 'usage'">
+          <div flex="100" flex-gt-sm="{{fs.panel ? 100 : 33}}" iiing-hide="fs && fs.panel != 'usage'">
```

Comment:
Sorry, the changes in client/components/registrations/registrations.directive.js and in client/components/locations/dashboard/_index.html are not attended by me. They came behind my back with some pull from master. I didn't notice them yesterday.